src/components/SelectInput/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import {
+    _cs,
+    listToMap,
+} from '@togglecorp/fujs';
+
+import SelectInputContainer from '../SelectInputContainer';
+import { rankedSearchOnList } from '../../utils';
+
+import styles from './styles.css';
+
+type OptionKey = string | number;
+interface Option {
+    [index: string]: any;
```

Comment:
No no no no

Reply:
It literally can be anything

Reply:
It's not about using any but that Option should instead just be T extends object